data/themes/darktable.css

```diff
@@ -1272,15 +1272,24 @@ dialog scrollbar
   margin: 0.35em 1.05em 1.05em 1.05em;
 }
 
-#preferences_box #shortcut_controls
+#preferences_box #shortcut_controls .search
 {
-  margin-left: 0.35em;
+  margin-right: 0.35em;
+}
+
+#preferences_box #shortcut_controls entry
+{
+  min-width: 220px;
```

Comment:
wouldn't this be better expressed in em to work ok on HiDPI screen?

Reply:
I have hesitated but you're right. Better with em. Fixed with last update.